### PR TITLE
Putting everything together on landing page first version

### PR DESCRIPTION
**Pull request by zhunan123**

### CODE REVIEW

**Comment by zhunan123** [comment]
I changed some styles and spacing to make the page look consistent to the design. Please let me know any where need to adjust I will change it accordingly. 




https://github.com/GayaHealth/Gaya_Website/assets/47227727/c9d59f15-db68-4b3c-98b0-238e2da92c3a

File: client/package.json [hunk 1]
```diff
@@ -5,8 +5,9 @@
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
-    "@mui/icons-material": "^5.13.7",
-    "@mui/material": "^5.13.7",
+    "@material-ui/core": "^4.12.4",
```
Comment (lawrenceslng) [inline client/package.json:8]: When I ran this I got an error stating `module not found`, and from npm it seems this package is deprecated: https://www.npmjs.com/package/@material-ui/core; we can remove this line and have the lines of code using this library to use the `@mui/material` instead

File: client/src/components/Root/SecondSection.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import {
     Typography,
     Button 
 } from '@material-ui/core';
```
Comment (lawrenceslng) [inline client/src/components/Root/SecondSection.js:6]: use `from '@mui/material'` instead

**Review by lawrenceslng — COMMENTED** [review]
left comments on the @material-ui/core pacakage

File: client/package.json (hunk 1 above)
Comment (zhunan123) [reply client/package.json:8]: yep, will update that.

**Review by lawrenceslng — APPROVED** [review]
good

**Pull request merged**